src/commands/registry.rs

```diff
@@ -5,6 +5,8 @@ use indicatif::{ProgressBar, ProgressStyle};
 use spin_oci::Client;
 use std::{io::Read, path::PathBuf, time::Duration};
 
+use super::new::ParameterValue;
```

Comment:
This type is used for template parameters.  I realise that today it happens to have the right shape and parsing behaviour that you need for annotations, but we should not bind the two together.  We should either:

* Create a new type defined by shape and behaviour (a la `EqualsSeparatedStringPair`) in a shared location, and use that in both places; or
* Create an Annotation type in this module, and use that here.

Reply:
I found a common type at `use spin_common::arg_parser::parse_kv`, so I've rewritten to code to use it.